### PR TITLE
Add "itemProperties" property that will apply to WMS layers.

### PR DESCRIPTION
**Pull request by stevage**

### CODE REVIEW

**Comment by kring** [comment]
Looks good.  I'm going to nitpick a few stylistic things, not because they're critically important but because a consistent style has a bit of value

File: src/Models/WebMapServiceCatalogGroup.js [hunk 1]
```diff
@@ -65,7 +65,14 @@ var WebMapServiceCatalogGroup = function(application) {
      */
     this.titleField = 'title';
 
-    knockout.track(this, ['url', 'dataCustodian', 'parameters', 'blacklist', 'titleField']);
+    /**
+     * Gets or sets a hash of properties that will be set on each child item.
+     * For example, { 'treat404AsError': false }
+     */
+
```
Comment (kring) [inline src/Models/WebMapServiceCatalogGroup.js]: Get rid of this blank line.

File: src/Models/WebMapServiceCatalogGroup.js [hunk 2]
```diff
@@ -323,6 +330,13 @@ function createWmsDataSource(wmsGroup, layer, supportsJsonGetFeatureInfo, suppor
         result.description += layer.Abstract;
     }
 
+    if (typeof(wmsGroup.itemProperties) === "object") {
```
Comment (kring) [inline src/Models/WebMapServiceCatalogGroup.js]: We use single quotes for strings throughout.  Also, the parenthesis after typeof are unnecessary.

**Comment by hilarycinis** [comment]
Thankyou :)

“…because a consistent style has a bit of value"

---

The information in this e-mail may be confidential and subject to legal professional privilege and/or copyright. National ICT Australia Limited accepts no liability for any damage caused by this email or its attachments.

**Comment by kring** [comment]
I'm talking about the style of the source code there.  I'd say consistent style of the UI has a lot more than a "bit" of value. :)

**Pull request merged**